examples/express-all-interactions/README.md

```diff
@@ -1,29 +1,42 @@
-# Express All Interacttions Example
+# Express All Interactions Example
 
 This example shows a fully functioning app using the
 [Slack Interactive Messages](https://github.com/slackapi/node-slack-interactive-messages) package.
 
 ## Setup
 
-### Create a Slack app
+### Create and Install a Slack app
 
 1. Create an app at <https://api.slack.com/apps> (don't close this window, you'll need it later).
 
+2. Once you've created the app, navigate to the "Install App" page on the sidebar. You'll see a hyperlink to add a 'permission scope' -- click on this.
+
+3. Scroll down to **Scopes**, and under the dropdown labeled **Select Permission Scopes**, select the option that says "Add slash commands and add actions to messages (and view related content)." Save your changes.
```

Comment:
i'm not sure i like the additional work of explicitly choosing a scope. slack shortcuts this by populating it for the developer after you chose to make a slash command. this seems like its a less than optimal solution for the problem we're trying to fix - the `SLACK_ACCESS_TOKEN` missing by the time you run the code. i think moving "Enable Interactive Components" and "Create a Slash Command" sections to before the "Run the code" section would resolve this without the need for new steps.